script/bootstrap

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+#
+# Idempotently create
```

Comment:
dope. so i have a docker-compose for simple local dev setup in this repo: https://github.com/tengen-io/tengen-io. I wonder how we can more seamlessly integrate DB setup with that?